net-p2p/go-ipfs-bin/go-ipfs-bin-0.4.15.ebuild

```diff
@@ -25,5 +25,21 @@ QA_PREBUILT="/usr/bin/ipfs"
 src_install() {
 	dobin ipfs
 
+	systemd_dounit "${FILESDIR}/ipfs.service"
+	newinitd "${FILESDIR}/ipfs.init" ipfs
+	newconfd "${FILESDIR}/ipfs.confd" ipfs
+
+	keepdir /var/log/ipfs
+	fowners -R ipfs:ipfs /var/log/ipfs
```

Comment:
```
>>> Install go-ipfs-bin-0.4.15 into /tmp/portage/net-p2p/go-ipfs-bin-0.4.15/image/ category net-p2p
chown: invalid user: ‘ipfs:ipfs’
```

In other words, you're creating the user *after* this. Also, please use `diropts` instead of `fowners` post creation.

You have two options:
1. Cheaper: create the user in `pkg_setup`.
2. Smarter: forget the above about `diropts`, create the user and do `fowners` in `pkg_preinst`. Then the user won't be needed when building a binary package but will be present for install.

Reply:
Second option seems also more nice  :)